pandas/tests/plotting/test_boxplot_method.py

```diff
@@ -254,6 +256,53 @@ def test_specified_props_kwd(self, props, expected):
 
         assert result[expected][0].get_color() == "C1"
 
+    def test_xlabel_ylabel(self):
```

Comment:
Could you split this into 3 distinct tests and use `pytest.mark.parameterize` over `vert`?

Reply:
Done.